packages/mdc-dialog/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@material/dialog",
+  "version": "0.1.0",
```

Comment:
Per our [file structure section in authoring-components.md](https://github.com/material-components/material-components-web/blob/master/docs/authoring-components.md#file-structure), the `version` of any new package should be set to `0.0.0`. This should also be updated within the `material-components-web` `package.json` under the `dependencies` section.